e2e/helpers/setupTeardown.ts

```diff
@@ -0,0 +1,83 @@
+import { Browser } from '@playwright/test'
+import { StorageStatePaths } from '../fixtures/authenticatedUserPages'
+import ACCESS_TYPE from './ACCESS_TYPE'
+import { addUserToEntityACL } from './acl'
+import {
+  createProject,
+  deleteFileHandleWithRetry,
+  deleteProject,
+  generateEntityName,
+} from './entities'
+import {
+  getAccessTokenFromCookie,
+  getAdminPAT,
+  getUserIdFromLocalStorage,
+} from './testUser'
+import { Project } from './types'
+import { UserPrefixes } from './userConfig'
+
+export const setupProject = async (
+  browser: Browser,
+  projectCreator: UserPrefixes,
```

Comment:
Not super important but I think `UserPrefixes` should be renamed to `UserPrefix` since it only refers to one prefix